buildchain/buildchain/vagrant.py

```diff
@@ -38,55 +39,62 @@ def mkdir_dot_vagrant() -> None:
         'targets': [constants.VAGRANT_SSH_KEY_PAIR],
         'uptodate': [True],
     }
-
-def task__vagrant_up_noprov() -> types.TaskDict:
-    """Run `vagrant up` without provisioning a development environment."""
-    cmd = [config.VAGRANT, 'up']
+    
+def _build_vagrant_cmd(self) -> List[str]:
+    cmd = [config.CMD['vagrant'], 'up']
```

Comment:
`s/config.CMD['vagrant']/config.ExtCommand.VAGRANT.value/`, same for the other occurrence in the file.